select_screen.lua

```diff
@@ -1351,8 +1351,15 @@ function select_screen.main()
     if cursor_data[1].state.ready and select_screen.character_select_mode == "1p_vs_yourself" then
       GAME.match = Match("vs", GAME.battleRoom)
       P1 = Stack(1, GAME.match, true, cursor_data[1].state.panels_dir, cursor_data[1].state.level, cursor_data[1].state.character)
+      if GAME.battleRoom.trainingModeSettings then
+        GAME.match.attackEngine = AttackEngine(P1)
+        local startTime = 200
```

Comment:
```suggestion
        local startTime = 350
```
I also believe we should give a little more time to get ready